src/app/technical-record/test-history/test-history.component.ts

```diff
@@ -8,28 +9,23 @@ import {TechRecordHelpersService} from '@app/technical-record/tech-record-helper
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class TestHistoryComponent implements OnInit {
+  @Input() testResultJson: TestResultModel[];
 
-  @Input() testResultJson: any;
+  constructor(public techRecHelpers: TechRecordHelpersService) {}
```

Comment:
we should remove this service when we can `TechRecordHelpersService`. not much significant